andy/src/main/java/nl/tudelft/cse1110/andy/config/SecureExamRunConfiguration.java

```diff
@@ -30,13 +26,14 @@ public Mode mode() {
     }
 
     @Override
+    @SuppressWarnings("DoubleBraceInitialization")
     public Map<String, Float> weights() {
-        return new HashMap<>(ImmutableMap.of(
```

Comment:
I would expect this to break with errorprone! `ImmutableMap.of(...)` is how it should be now, I think!